src/recipes.jl

```diff
@@ -7,18 +7,23 @@ export partitionplot
     args
 end
 
-@recipe function f(o::PartitionPlot; nparts = 100, stat = nothing, by = nothing)
+getvalue(x) = x 
+getvalue(x::DataValues.DataValue) = get(x)
+
+@recipe function f(o::PartitionPlot; nparts = 100, stat = nothing, by = nothing, dropmissing = false)
     t = o.args[1]
     sel_x = o.args[2] 
     stat = (stat == nothing) ? Extrema() : stat
     if length(o.args) == 3
         sel_y = o.args[3]
         T = fieldtype(eltype(t), IndexedTables.colindex(t, sel_x))
+        s = dropmissing ? 
+            series(IndexedPartition(T, stat, nparts); filter = x->any(!isnull, x), transform = x->map(getvalue,x)) :
```

Comment:
Sorry to insist so much on this line of code.. But shouldn't you check that both `sel_x` and `sel_y` values are there, meaning `filter = x->all(!isnull, x)` ?

Reply:
Again you're right.  I'm putting together some interactive tests to check all this.